admin_guide/node-allocatable-resources.adoc

```diff
@@ -0,0 +1,58 @@
+= Node Allocatable resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node has a chance to specify how much resources get consumed
+by kubernetes related components (e.g. kubelet, kube-proxy, docker) running on a node
+and by remaining non-kubernetes components (e.g. system daemons, databases).
+Once specified, scheduler has more information about how much resources
+(e.g. memory, cpu) a node has allocated for pods.
+
+== Setting reserved resources
+
+Both kubernetes and non-kubernetes components reserved resources are specified through kubelet command-line flags:
```

Comment:
we should show these flags as they would be provided in the `node-config.yaml` file for openshift-node.